libs/core/src/main/java/org/opensearch/core/common/util/CollectionUtils.java

```diff
@@ -92,7 +102,13 @@ public static <T> List<T> rotate(final List<T> list, int distance) {
     }
 
     /**
-     * in place de-duplicates items in a list
+     * In place de-duplicates items in a list<p>
```

Comment:
This is weird, we still have unclosed `<p>` tag but there is no error ...

Reply:
> This is weird, we still have unclosed `<p>` tag but there is no error ...

This is javadoc 😄 
unclosed <p> are just line break.
The error i had previously was because you transformed my "TODO" (text) into a "@throws" (not text). That was, of course, the right thing to do. But we ended up with a <p> with no text after it.

the check is being pedantic on this while missing tons of truly very javadoc.
I ran an inteliJ javadoc inspection and fixed tons of them already (for the next PR).